utils/oscap-oval.c

```diff
@@ -217,6 +224,9 @@ int app_collect_oval(const struct oscap_action *action)
 	struct oval_generator		*generator = NULL;
 	int ret = OSCAP_ERROR;
 
+	/* Turn on verbosity */
+	oscap_set_verbose(action->verbosity_level, action->f_verbose_log, false);
```

Comment:
Correct setting of verbosity level is checked by a function `check_verbose_options`. It is called during options processing. If a user runs `oscap` with verbosity level wrongly specified, he will get an error message and oscap will terminate.

However, we should wrap this by `if goto cleanup` to terminate when the output file can't be opened.